pkg/scaffold/scaffoldtest/scaffoldtest.go

```diff
@@ -18,12 +18,12 @@ package scaffoldtest
 
 import (
 	"bytes"
+	"go/build"
 	"io"
 	"io/ioutil"
 	"os"
 	"path"
 	"path/filepath"
-	"go/build"
 
```

Comment:
I do not think that is required and if yes, then IHMO should be done across the repo and not just one file.

Reply:
Alphabetical order should be followed in each group that you mentioned.

And it is repo-wide, haven't find any other in the whole repo.

Reply:
Are you using a linter to get it? If yes, could you please provide this info as well?